src/App.js

```diff
@@ -1,26 +1,9 @@
-import logo from './logo.svg';
-import './App.css';
+import Calculator from './components/Calculator';
 
 function App() {
   return (
```

Comment:
- [ ] The `App.js` component should be class-based as this is part of the project requirements.